tests/testdata/demo-modules/demo-module-3/load.php

```diff
@@ -2,7 +2,6 @@
 /**
  * Module Name: Demo Module 3
  * Description: This is the description for demo module 3.
- * Focus: images
```

Comment:
See above, this file still needs to be moved accordingly.

Reply:
👍🏼